easybuild/easyconfigs/s/SIP/SIP-4.16.4-goolf-1.5.14-Python-2.7.6.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'SIP'
+version = '4.16.4'
+
+homepage = 'http://www.riverbankcomputing.com/software/sip/'
+description = """
+SIP is a tool that makes it very easy to create Python bindings for C and C++ 
+libraries. It was originally developed to create PyQt, the Python bindings 
+for the Qt toolkit, but can be used to create bindings for any C or C++ library. 
+"""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'} 
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://sourceforge.net/projects/pyqt/files/sip/sip-4.16.4']
+
+python = 'Python'
+pythonver = '2.7.6'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+dependencies = [(python, pythonver)]
+
+preconfigopts = 'mkdir -p %(installdir)s/bin %(installdir)s/lib/python2.7/site-packages/SIP %(installdir)s/include %(installdir)s/sip && '
+configopts = " configure.py -b %(installdir)s/bin -d %(installdir)s/lib/python2.7/site-packages/SIP -e %(installdir)s/include -v %(installdir)s/sip" 
```

Comment:
avoid too long lines, use `configopts +=` to break it up

don't hardcode `2.7`, use `pythonshortver` and escape `%` where you need to (`%(installdir)s`)